spec/polaris-management-service.yml

```diff
@@ -917,6 +917,7 @@ components:
             - OAUTH
             - BEARER
             - SIGV4
+            - IMPLICIT
```

Comment:
Please add a brief note to the `New Features` section in CHANGELOG.md about this.

Reply:
Done.